### ISSUE TITLE
About mAP

### ISSUE THREAD

**Issue by zwyyy215** [ppengtang/pcl.pytorch#3]
According to the operation of read me file, the value of mAP is 47.8%, which is a little different from what the author said. Is this normal? Can I adjust the parameters to make it larger?

**Comment by ppengtang** [comment]
Hi,

Thanks for your question.
I have run the PASCAL VOC 2007 experiment on my own machine multiple times using the parameters provided in the repo. Due to the fixed random seed, all these times I got the same experimental results: mAP 49.2% and CorLoc 65.0%. I'll provide a trained model soon.
You can also feel free to change the parameters such as the random seed (__C.RNG_SEED).
By the way, what is the CorLoc number you got?

**Comment by zwyyy215** [comment]
I can't find the discovery.pkl. That's weird.

**Comment by ppengtang** [comment]
Have you run the command `python tools/test_net.py --cfg configs/baselines/vgg16_voc2007.yaml \
  --load_ckpt Outputs/vgg16_voc2007/$MODEL_PATH \
  --dataset voc2007trainval`?

**Comment by zwyyy215** [comment]
Yes.I find the detections.pkl and calculated the mAP(47.8%).
However,I can't find the discovery.pkl.

**Comment by ppengtang** [comment]
Are you sure that you set --dataset to voc2007trainval instead of voc2007test? If you set --dataset to voc2007trainval, there should be a discovery.pkl

**Comment by zwyyy215** [comment]
You're right.Thank you very much!

**Comment by zwyyy215** [comment]
I forgot to tell you the CorLoc number！
Mean CorLoc = 0.6397
Mean AP = 0.4780

**Comment by ppengtang** [comment]
Hi,
The model trained on PASCAL VOC 2007 is available on [Baidu Yun](https://pan.baidu.com/s/1BK1STmTOcLypyz1HPIhf_Q) (please use code bch4).

**Comment by Mickeybaby** [comment]
I have the same problem.
The two evaluation values are about 1.2% lower than the results of the paper.
I tried to change the random number.（__C.RNG_SEED.But it didn't work.
Do you have any suggestions for improving the results?

**Comment by ppengtang** [comment]
Hi, I'm not sure about the reason. In my experiments, I can always get the same 49.0% mAP.
Maybe you can download my trained model [here](https://pan.baidu.com/s/1BK1STmTOcLypyz1HPIhf_Q). #5
